### PR TITLE
Adding missing Close method for HttpWebResponse

### PR DESCRIPTION
**Pull request by tijoytom**
@stephentoub @davidsh

### CODE REVIEW

File: src/System.Net.Requests/src/System/Net/HttpWebResponse.cs [hunk 1]
```diff
@@ -354,6 +354,11 @@ public string GetResponseHeader(string headerName)
             return ((headerValue == null) ? String.Empty : headerValue);
         }
 
+        public override void Close()
+        {
+            this.Dispose(true);
```
Comment (davidsh) [inline src/System.Net.Requests/src/System/Net/HttpWebResponse.cs]: nit: CoreFx coding guidelines say to remove uses of `this.`.

**Comment by tijoytom** [comment]
@karelz FYI

File: src/System.Net.Requests/src/System/Net/HttpWebResponse.cs [hunk 2]
```diff
@@ -144,6 +144,12 @@ public virtual CookieCollection Cookies
                 CheckDisposed();
                 return _cookies;
             }
+
+            set
+            {
+                CheckDisposed();
+                _cookies = value;
+            }
```
Comment (stephentoub) [inline src/System.Net.Requests/src/System/Net/HttpWebResponse.cs]: Is there a test for this?

Comment (tijoytom) [inline src/System.Net.Requests/src/System/Net/HttpWebResponse.cs]: @stephentoub I thought the get covered it , but it did not , modified the get to set first and check.

**Comment by tijoytom** [comment]
@davidsh @stephentoub 
Is this good for merge? 

Infra failure from Ubuntu
Build timed out (after 120 minutes). Marking the build as aborted.

**Pull request merged**